recipes/winregistry/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "winregistry" %}
+{% set version = "1.1.1" %}
+{% set python_min = "3.7" %}
```

Comment:
This variable should only be set if you need 3.10 or newer.
```suggestion
```

Reply:
removed it :)